COLLABORATOR_GUIDE.md

```diff
@@ -603,17 +603,15 @@ like this:
 To https://github.com/nodejs/node
  ! [rejected]              master -> master (fetch first)
 error: failed to push some refs to 'https://github.com/nodejs/node'
-hint: Updates were rejected because the remote contains work that you do
-hint: not have locally. This is usually caused by another repository pushing
-hint: to the same ref. You may want to first integrate the remote changes
-hint: (e.g. 'git pull ...') before pushing again.
+hint: Updates were rejected because the tip of your current branch is behind
+hint: its remote counterpart. Integrate the remote changes (e.g.
```

Comment:
The new description is definitely on the point and correct but the former one seems easier to understand or at least to me, it seems the former version required less knowledge about git. I personally would stick to the former version for the first sentence.

Reply:
How about just removing all the `hint` text and leaving lines 603-605? That's the relevant part anyway. The hint text will be whatever and we don't control it.

Reply:
Looking at it again: I just realized this is 1-to-1 the actual git hint. Therefore it's probably indeed good to use that.